templates/endpoints_v2/AdvancedModelConfigs.md

```diff
@@ -0,0 +1,63 @@
+# Advanced Model Configuration
+
+Each model is defined by a YAML configuration file in the `models` directory.
+
+## Modify an existing model
+
+To modify an existing model, simply edit the YAML file for that model.
+Each config file consists of three sections: 
+
+- `deployment_config`, 
+- `engine_config`, 
+- `scaling_config`.
+
+It's best to check out examples of existing models to see how they are configured.
+
+## Deployment config
+
+The `deployment_config` section corresponds to
+[Ray Serve configuration](https://docs.ray.io/en/latest/serve/production-guide/config.html)
+and specifies how to [auto-scale the model](https://docs.ray.io/en/latest/serve/scaling-and-resource-allocation.html)
+(via `autoscaling_config`) and what specific options you may need for your
+Ray Actors during deployments (using `ray_actor_options`). We recommend using the values from our sample configuration files for `metrics_interval_s`, `look_back_period_s`, `smoothing_factor`, `downscale_delay_s` and `upscale_delay_s`. These are the configuration options you may want to modify:
```

Comment:
If I'm a new user, I don't know what `Ray Actors` are and its relationship to the model or deployment.